generate/templates/ping360.py.in

```diff
@@ -163,29 +159,32 @@ if __name__ == "__main__":
 
     print("full scan in %dms, %dHz" % (1000*(tend_s - tstart_s), 400/(tend_s - tstart_s)))
 
-    # turn on auto-scan with 1 grad steps
-    p.control_auto_transmit(0,399,1,0)
+    if p.firmware_version > (3, 1, 1):
```

Comment:
This should be a check against protocol version (`>= 1.1.0`), not firmware version (different devices supporting the protocol may have different firmware versions).